Contents/Code/__init__.py

```diff
@@ -364,22 +364,22 @@ def name_to_initials(self, input_name):
         # Only the surname stays as whole, the rest gets truncated
         # and merged with dots.
         # Example: 'Arthur Conan Doyle' -> 'A.C.Doyle'
-        nameparts = input_name.split()
-        newName = ""
+        name_parts = input_name.split()
+        new_Name = ""
         
         # Check if prename and surname exist, otherwise exit
-        if len(nameparts) < 2:
+        if len(name_parts) < 2:
             return input_name
         
         # traverse through prenames
-        for i in range(len(nameparts)-1):
-            s = nameparts[i]
+        for i in range(len(name_parts)-1):
+            s = name_parts[i]
             # If prename already is an initial take it as is
-            newName += (s[0] + '.') if len(s)>2 and s[1]!='.' else s
+            new_Name += (s[0] + '.') if len(s)>2 and s[1]!='.' else s
```

Comment:
Last thing, I promise! You left the second word capitalized, should be `new_name`.

Reply:
No worries, done ;)